Quantum-Control-Applications/Optically addressable spin qubits/NV center in a confocal setup/04_calibrate_delays.py

```diff
@@ -98,6 +99,7 @@
 
     with stream_processing():
         # Directly derive the histograms in the stream processing
+        # NOTE: use `2 * meas_len` for QOP version >= 3.5.0 because time bins are 0.5ns
```

Comment:
Same consideration here, I would actually always set the resolution defined above in ns, so that users won't need to think about it in the plotting section, as having a different unit depending whether you are on QOP350 or below is confusing.